.github/workflows/ci.yaml

```
@@ -16,7 +17,8 @@ jobs:
 
     steps:
     - uses: actions/checkout@v4
-
+      with:
+          fetch-depth: 0
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)